docs/ignore.md

```diff
@@ -24,6 +24,27 @@ coverage
 
 It’s recommended to have a `.prettierignore` in your project! This way you can run `prettier --write .` to make sure that everything is formatted (without mangling files you don’t want, or choking on generated files). And – your editor will know which files _not_ to format!
 
+By default prettier ignores files in version control systems directories (".git", ".svn" and ".hg") and `node_modules` (if [`--with-node-modules` CLI option](cli.md#--with-node-modules) not specified)
+
+So by default it will be
+
+```js
+**/.git
+**/.svn
+**/.hg
+**/node_modules
+```
+
+and
+
+```js
+**/.git
+**/.svn
+**/.hg
+```
+
+if [`--with-node-modules` CLI option](cli.md#--with-node-modules) provided
```

Comment:
Missing full stop at the end of the sentence. It might be easier to read this statement if we put this line before the second block:

````txt
and if [`--with-node-modules` CLI option](cli.md#--with-node-modules) is provided:

```txt
...
```
````